index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+const _ = require('lodash');
+
+// object representing a robots.txt that blocks everything:
+const disallowAll = {
+  '*': ['/']
+};
+
+const defaults = {
+  // can be over-ridden by plugin users:
+  userAgents: disallowAll,
```

Comment:
I wonder if we should do something like this:

```javascript
envs: {
  production: {
  
  },
  stage: {
  },
  '*': { //this could act as an else or wildcard
  }
}
```